src/item.h

```diff
@@ -327,13 +327,21 @@ class item : public visitable<item>
     /**
      * Returns the default color of the item (e.g. @ref itype::color).
      */
+
     nc_color color() const;
     /**
      * Returns the color of the item depending on usefulness for the player character,
      * e.g. differently if it its an unread book or a spoiling food item etc.
      * This should only be used for displaying data, it should not affect game play.
      */
+
     nc_color color_in_inventory() const;
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    std::string sname( unsigned int quantity = 1 ) const;
```

Comment:
A comment what this function does and what the difference to `tname` (very similar name!) is would be nice.

Reply:
That function may not even be kept. It was part of trying to fix a sorting issue (short for 'sorting name') but I'm not sure if that's really within the scope of these changes.